GetRCConfiguration/__tests__/handler.test.ts

```diff
@@ -14,10 +16,18 @@ import {
   handleEmptyErrorResponse
 } from "../handler";
 import { IConfig } from "../../utils/config";
+import { redisClientMock } from "../../__mocks__/redis.mock";
 
 const aUserId = "aUserId" as NonEmptyString;
 const aConfig = { INTERNAL_USER_ID: "internalUserId" } as IConfig;
 
+const getTaskMock = jest.fn();
+jest.spyOn(redis_storage, "getTask").mockImplementation(getTaskMock);
```

Comment:
why not spying also on setTask to verify that if the value is not in cache it will be cached?

Reply:
[4ee62e6](https://github.com/pagopa/io-functions-service-messages/pull/57/commits/4ee62e697113bb7c183363c2c586eec3a68ca1f7)